tests/app/clients/test_twilio.py

```diff
@@ -599,19 +599,43 @@ def test_send_sms_twilio_callback(
         assert response_dict['sid'] == twilio_sid
 
 
+@pytest.mark.parametrize(
+    (
+        'response_dict',
+        'exception',
+    ),
+    (
+        (
+            {
+                'code': 21606,
+                'message': "The 'From' phone number provided (+61412345678) is not a valid message-capable Twilio phone number for this destination",
+            },
+            InvalidProviderException,
+        ),
+        (
+            {
+                'code': 21617,
+                'message': 'Unable to create record: The concatenated message body exceeds the 1600 character limit',
+            },
+            NonRetryableException,
+        ),
+    ),
+    ids=(
+        'invalid-from-number',
+        'message-too-long',
+    ),
+)
 def test_send_sms_raises_invalid_provider_error_with_invalid_twilio_number(
```

Comment:
you've parametrized a test whose name is specific to invalid provider. suggest changing the name to something more generic or not parametrizing.